packages/runtime/src/next/api-handler.ts

```diff
@@ -148,8 +151,23 @@ export function MakeswiftApiHandler(
         )
       }
 
+      case 'fonts': {
+        const fonts = (await getFonts?.()) ?? []
+        return res.json(fonts)
+      }
+
       default:
         return res.status(404).json({ message: 'Not Found' })
     }
   }
 }
+
+type Fonts = Font[]
+
+type Font = {
+  family: string
+  variants: FontVariant[]
+  fallback?: string[]
+}
+
+type FontVariant = { weight: string; style: 'italic' | 'normal'; src?: string }
```

Comment:
nit: Can we put these types together with the other types instead of at the end of the file?

Reply:
Updated!